Micropython IDE clarification (#662)
arduino/docs-content: content/tutorials/generic/micropython-installation/micropython-installation.md

### Installing the Core

> Start by making sure that you have the respective core installed. For now, wait with the new Arduino Lab for MicroPython, instead open the usual Arduino IDE and navigate to the boards manager. Search for your board, and make sure you have the latest version of the core installed. 

Reviewer: ```suggestion
Start by making sure that you have the respective core installed. Open Arduino IDE (not Arduino Lab for MicroPython) and navigate to the boards manager. Search for your board, and make sure you have the latest version of the core installed. 
```

The previous wording was confusing.

2. Connet the `REC` pin to ground to expose the boards file system to your computer, this is easier if you put the board in a breadboard.
3. Drag and drop the firmware file onto the boards flash storage which shows up as an external storage device on your computer, and you're done!

Go to the Arduino Lab MicroPython IDE and press connect in the top left, choose the port, and you are now ready to program your Nano RP2040 Connect.

## Arduino Portenta H7

> To install MicroPython on the Portenta H7 you will need to install the corresponding core which in turn installs the required command line tool. For now, wait with the new Arduino Lab for MicroPython, instead open the usual Arduino IDE and navigate to the boards manager. Search for "Portenta H7", and make sure you have the latest version of the core installed.

Reviewer: ```suggestion
To install MicroPython on the Portenta H7 you will need to install the corresponding core which in turn installs the required command line tool. Open Arduino IDE (not Arduino Lab for MicroPython) and navigate to the boards manager. Search for "Portenta H7", and make sure you have the latest version of the core installed.
```
